test/e2e/files/containerd.service

```diff
@@ -19,7 +19,7 @@ After=network.target local-fs.target
 
 [Service]
 ExecStartPre=-/sbin/modprobe overlay
-ExecStart=/usr/bin/containerd
+ExecStart=/usr/local/bin/containerd
```

Comment:
What do you think about pulling the unit file from https://raw.githubusercontent.com/containerd/containerd/main/containerd.service as described in the containerd installation document in  https://github.com/containerd/containerd/blob/main/docs/getting-started.md#systemd?

Reply:
Hmm... That is otherwise identical, but it seems to lack the `LimitNOFILE=infinity` setting. Probably shouldn't cause any problems though.

Reply:
Uhmmkay. I think I'll leave it as it is for now. It does not look like worth the effort. That service file is not even versioned, it is behind a static URL. So if it ever need to be changed/updated, it can't be handled like this.

Reply:
Okay, after checking that for CRI-O the service file is both part of the release tarball and available in the source tree, I changed the playbook after all to use those for CRI-O and use the above-mentioned URL for containerd.

Reply:
One benefit is that there is very less (or almost none) risk of having incorrect configuration like right now. But it is not something must to have.